test/component_test.go

```
@@ -0,0 +1,150 @@
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type validationOption struct {
+	DomainName          string `json:"domain_name"`
+	ResourceRecordName  string `json:"resource_record_name"`
+	ResourceRecordType  string `json:"resource_record_type"`
+	ResourceRecordValue string `json:"resource_record_value"`
+}
+
+type zone struct {
```

golangci-lint (1.62.2)

type `zone` is unused

(unused)